### PR TITLE
LPD-35743 | Test Fix | Poshi >>> Playwright

### PR DESCRIPTION
**Pull request by rafaellajordao**
### Motivation:
Test failing due to Poshi limitations.

### Proposed Solution:
Transcribe test to playwright in order to facilitate maintenance.

**More details:** [LPD-35743](https://liferay.atlassian.net/browse/LPD-35743)

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:analytics-cloud

**Comment by rafaellajordao** [comment]
> **Jenkins Build:**[test-portal-acceptance-pullrequest(master)#12507](https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/12507/)
> **Jenkins Report:**[jenkins-report.html](https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/12507/jenkins-report.html)
> **Jenkins Suite:**analytics-cloud
> **Pull Request:**[liferay-ac#1833](https://github.com/liferay-ac/liferay-portal/pull/1833)
> **Testray Routine:**[EE Pull Request](https://testray.liferay.com/#/project/630146/routines/630264)
> **Testray Build:**[[master] ci:test:analytics-cloud - rafaellajordao > liferay-ac - PR#1833 - 2024-09-09[13:55:57]](https://testray.liferay.com/#/project/630146/routines/630264/build/5942137)
> **Testray Build ID:**5942137
> **Testray Importer:**[publish-testray-report#27389](https://test-1-27.liferay.com/job/publish-testray-report/27389/)

the test created is passing correctly! :smile:

File: modules/test/playwright/tests/osb-faro-web/page.spec.ts [hunk 1]
```diff
@@ -563,6 +619,87 @@ test(
...
+				page,
+				pageName: 'Path',
+			});
+		});
+
+		await test.step('Check that Snúið Vinsælar þú Page appear as referral page', async () => {
+			await expect(
+				page.getByText('Snúið Vinsælar ...', {exact: true}).first()
+			).toBeVisible({
+				timeout: 100 * 1000,
```
Comment (marcosapmf) [inline modules/test/playwright/tests/osb-faro-web/page.spec.ts:689]: @rafaellajordao , can you remove this timeout? if I'm no wrong, the default timeout for `toBeVisible` is 15s. I think it is enough time :smile: 
Also, if you look for that in the code, you'll see the other teams are avoiding to use it in this way

**Review by marcosapmf — CHANGES_REQUESTED** [review]
Thanks @rafaellajordao, please see my comment above

**Comment by rafaellajordao** [comment]
ci:test:sf

File: modules/test/playwright/tests/osb-faro-web/page.spec.ts (hunk 1 above)
Comment (rafaellajordao) [reply modules/test/playwright/tests/osb-faro-web/page.spec.ts]: of course! fixed! :smile:

**Comment by rafaellajordao** [comment]
ci:forward

**Pull request closed without merging**